### PR TITLE
Remove incorrect JS for indeterminate progress

### PR DESCRIPTION
**Pull request by FWDekker**
[The documentation for Pico](https://picocss.com/docs/progress.html) currently says you need to use JS to set a `<progress>` bar to an indeterminate state, but this doesn't actually work. The following code does _not_ create an indeterminate progress bar:

```html
<progress id="indeterminate-progress" value="25" max="100"></progress>
<script>
  document.getElementById('indeterminate-progress').indeterminate = true;
</script>
```

Based on the [MDN article about `<progress>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress), a progress bar is in an indeterminate state if there is no `value` attribute. This happens to be the case with the progress bar in the Pico documentation as well.

In other words, the JS in the Pico documentation doesn't do anything.

---

I have removed the unnecessary JS from the documentation, and documented that a progress bar is indeterminate if there is no `value` attribute.

### CODE REVIEW

**Comment by walterdavis** [comment]
The only element I know of for sure that does act this way is the checkbox. If you set its indeterminate property to true, then it will show the [-] icon instead of [ ] or [x]. This is handy for "check all" features, where the checkbox in question is both acting as a button to check all and a sort of state indicator for "are all of these checkboxes checked?".

**Comment by FWDekker** [comment]
@walterdavis True. [mdn confirms](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox) that you need JS to set the indeterminate state for a checkbox. It looks like that case is [already covered by Pico's documentation](https://picocss.com/docs/forms.html).

This PR only covers the indeterminate state for progress bars, which is actually unrelated to the `indeterminate` property. Instead, a progress bar is indeterminate if and only if the `value` attribute is not set in the DOM.

**Comment by lucaslarroche** [comment]
Hi @FWDekker,
Thank you, good find.
I will merge your PR into `dev`. Then I will add a few more fixes before merging into `master`.
